windows-packaging/CalicoWindows/config.ps1

```diff
@@ -59,7 +59,8 @@ $env:CNI_IPAM_TYPE = "calico-ipam"
 $env:VXLAN_VNI = "4096"
 # Prefix used when generating MAC addresses for virtual NICs.
 $env:VXLAN_MAC_PREFIX = "0E-2A"
-
+# Network Adapter used on VXLAN, leave blank for primary NIC. 
+$env:VXLAN_ADAPTER = ""

```

Comment:
Does the default `""` value do something sensible (i.e. give us the existing behaviour)?

Reply:
Yes, `New-HNSNetwork` checks for the value, if it does not exists use the primary interface, what is the default value

https://github.com/microsoft/SDN/blob/b189fc96bcba8ea3ed6c77b51bca9266a8683ae7/Kubernetes/windows/hns.psm1#L208

Reply:
Yes, `New-HNSNetwork` checks for the value - https://github.com/microsoft/SDN/blob/b189fc96bcba8ea3ed6c77b51bca9266a8683ae7/Kubernetes/windows/hns.psm1#L208

If the value does not exist, the primary interface is used, giving us the current behavior.